cvl/cvl_test.go

```diff
@@ -361,25 +369,38 @@ func TestMain(m *testing.M) {
 	}
 
 	os.Exit(code)
+
+}
+
+var configDb *db.DB
+
+func init() {
+	configDb, _ = db.NewDB(db.Options{
+		DBNo:               db.ConfigDB,
+		InitIndicator:      "CONFIG_DB_INITIALIZED",
```

Comment:
Remove this "**InitIndicator**" option to fix pipeline test errors. It would be a fresh installation of redis on the build VM and CONFIG_DB_INITIALIZED would not have been set.
Also, do not ignore error returned by `db.NewDB()`. Add a panic.